.azure-pipelines/baseline_test/baseline.test.template.yml

```diff
@@ -10,8 +10,8 @@ jobs:
       TOPOLOGY: t0
       MIN_WORKER: $(T0_INSTANCE_NUM)
       MAX_WORKER: $(T0_INSTANCE_NUM)
-      KVM_IMAGE_BRANCH: "master"
-      MGMT_BRANCH: "master"
+      KVM_IMAGE_BRANCH: $(Build.SourceBranchName)
+      MGMT_BRANCH: $(Build.SourceBranchName)
```

Comment:
Maybe this parameter will not be effective

Reply:
It could be effective, pipeline was triggered, but didn't finish test because elastictest do not support branch in my own fork, here is the log:
```
+ git checkout --progress --force refs/remotes/origin/support_baseline_test_in_202411
error: pathspec 'refs/remotes/origin/support_baseline_test_in_202411' did not match any file(s) known to git
```

Then I changed it to 202411 and tested, all functions are good

Reply:
Can you create a pipeline and have a test? I'm still worrying about this as I remembered, I had a similar try and the parameter is not effective. In your new created pipeline, you can specify the yaml file from your own branch.

Reply:
Yes I created a pipeline and tested, I successfully created testplans, and because I'm using $(Build.SourceBranchName), the testplan was trying to checkout the branch in my own repo and reported above error, so I think it has proved the parameter is effective